examples/nodejs/Readme.md

```diff
@@ -0,0 +1,53 @@
+# Using this example
+
+To use this example you first need to install dependencies and build the code
```

Comment:
I don't know if it is a French thing, but I usually avoid saying "you" when explaining something...

Reply:
e.g : To use this example, install the dependencies and build the codebase

Reply:
Not using personal pronouns ("you", "we","I", "they", etc) makes text more formal. I don't have strong feelings on this matter but I tend to not use personal pronouns when recounting events in documentation but I think when providing instructions like in this case, using "you" is not as bad.